### PR TITLE
[Improve](Scan) add a session variable to make scan run serial

### PR DESCRIPTION
**Pull request by eldenmoon**
## Proposed changes

Parallel scanning can result in some read amplification, for example, select * from xx where limit 1 actually requires only one row of data. However, due to parallel scanning of multiple tablets, read amplification occurs, leading to performance bottlenecks in high-concurrency scenarios. This PR Adding a SessionVariable to enforce serial scanning can help mitigate this issue.

Issue Number: close #xxx

<--Describe your changes.-->

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by eldenmoon** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java [hunk 1]
```diff
@@ -531,6 +533,9 @@ public class SessionVariable implements Serializable, Writable {
     @VariableMgr.VarAttr(name = ENABLE_ODBC_TRANSCATION)
     public boolean enableOdbcTransaction = false;
 
+    @VariableMgr.VarAttr(name = ENABLE_RUN_SERIAL)
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java:536]: Please use `description` field to add document for this new variable

Comment (morningman) [reply fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java:536]: And better rename it to a more concrete one. `run serial` is too common

Comment (eldenmoon) [reply fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java]: done

**Comment by eldenmoon** [comment]
run buildall

**Review by xxiao2018 — APPROVED** [review]
LGTM

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**